packages/terra-date-time-picker/tests/jest/DateTimePicker.test.jsx

```diff
@@ -88,9 +88,9 @@ it('Should not throw any errors while date value is outside of the Min, Max rang
   const dateYearInput = datePicker.find({ name: 'terra-date-year-input', type: 'text' }).at(0);
   dateYearInput.simulate('change', { target: { value: '2017' } });
   expect(() => {
-    const hourInput = datePicker.find({ name: 'terra-time-hour-input' }).at(0);
+    const hourInput = datePicker.find({ name: 'terra-time-hour-input' }).at(2);
```

Comment:
@benbcai I couldn't use DIVs because then each input would be on a new line, or I would need new styles. Instead, I discovered that the 3rd node in this find() object is the input.  The first and 2nd nodes are a _function_ and a _host_. I think the function is the intl wrapper and the host is the frag? IDK. It works though.

Reply:
That makes sense and this change looks good to me.